quickstart/README.md

```diff
@@ -35,11 +35,16 @@ This document describes how to quickly get started with Hosting Control Planes a
 
 ## What it does
 1. Enables preview_hypershift
-2. Creates a `local-cluster` `managedCluster` for the OpenShift cluster you are installing to
-3. Imports the `local-cluster`
-4. Applies the Hosting Service Cluster addon (Hypershift) to the `local-cluster` (Hub)
+2. Creates a `local-cluster` `managedCluster` for the OpenShift cluster you are installing to if it doesn't
+3. Imports the `local-cluster` if it doesn't exist
+4. Applies the Hosting Service Cluster addon (Hypershift) to the `local-cluster` (Hub) if it doesn't exist
 
-## Use the hypershift CLI to create a HostedCluster and NodePool
+## Use the oc hcp or hypershift CLI to create a HostedCluster and NodePool
```

Comment:
Use uppercase for HCP since it's an acronym. 

hypershift - spell HyperShift with the correct cases